drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -262,6 +267,28 @@ local function level_attr_handler(driver, device, ib, response)
   end
 end
 
+local function level_min_attr_handler(driver, device, ib, response)
+  if ib.data.value ~= nil then
+    local level = math.floor((ib.data.value / 254.0 * 100) + 0.5)
+    device:set_field(LEVEL_MIN, level)
+    local level_max = device:get_field(LEVEL_MAX)
+    if level_max then
+      device:emit_event_for_endpoint(ib.endpoint_id, capabilities.switchLevel.levelRange({minimum = level, maximum = level_max}))
```

Comment:
Once we emit the event I think we could probably remove the persisted fields since we only need to temporarily remember one while we wait for the other.

Reply:
Made this change in https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1220/commits/c7bfc422f8876abcd8f5cbe746aeef3c5aabfbbc